.devcontainer/docker-compose.dev.yml

```diff
@@ -0,0 +1,12 @@
+version: '3'
+
+services:
+  web:
+    volumes:
+      - .:/app:delegated
+      - vscode-server:/root/.vscode-server:cached
+      - /var/run/docker.sock:/var/run/docker.sock:cached
```

Comment:
@jofsky This line is new. Does adding it solve a problem?

Reply:
No I don't believe so let me try removing see what happens.